third_party/eigen3/BUILD

```diff
@@ -48,3 +50,35 @@ filegroup(
     ),
     visibility = ["//tensorflow:__subpackages__"],
 )
+
+filegroup(
+    name = "eigen_third_party_header_files",
+    srcs = EIGEN3_THIRD_PARTY_HEADERS,
+    visibility = ["//visibility:public"],
+)
+
+genrule(
+    name = "install_eigen_headers",
+    srcs = [
+        "@eigen_archive//:eigen_header_files",
+        ":eigen_third_party_header_files",
+    ],
+    outs = ["include"],
```

Comment:
Ideally we should declare all output files as `outs`, but since it's only for installing headers to `/usr/include`, I guess it's fine. But please make sure no other targets depend on it, otherwise the build may not be hermetic.

Reply:
If you're fine with it then we'll just leave it for now. Nothing else uses those targets and if they did it would just be an extra copy of files in genfiles so i think it should still be hermetic.

I am thinking in the future maybe we can have more targets like this so handle installing everything (eg libs, headers, pkgconfig file etc). For now my ebuild installs manually so its not high priority.

Reply:
SGTM